filebeat/processor/add_kubernetes_metadata/matchers.go

```diff
@@ -77,7 +77,8 @@ const containerIdLen = 64
 const podUIDPos = 5
 
 func (f *LogPathMatcher) MetadataIndex(event common.MapStr) string {
-	if value, ok := event["log"].(common.MapStr)["file"].(common.MapStr)["path"]; ok {
+	value, err := event.GetValue("log.file.path")
+	if err == nil {
```

Comment:
I think it's most common in Go to return early on error and reduce the level of nesting. How about `if err != nil { return "" }`?

Reply:
Since return "" is just out of the if check bracket, I did not want to do an extra one. :-)